connection.js

```diff
@@ -1020,42 +1001,26 @@ class Connection {
         ////// end of *** UHMAEAT v3.1 : Universal Highly Modulable And Expandable Argv Tree ***
 
         function minmaxHandicapFamilyReject(argNameString) {
-            // first, we define rankedUnranked and minMax depending on argNameString
-            let rankedUnranked = "";
-            // if argNameString does not include "ranked" or "unranked", we keep default value for rankedunranked
-            if (argNameString.includes("ranked") && !argNameString.includes("unranked")) {
-                rankedUnranked = "for ranked games";
-            } else if (argNameString.includes("unranked")) {
-                rankedUnranked = "for unranked games";
-            }
-
-            let minMax = "";
-            let increaseDecrease = "";
-            if (argNameString.includes("min")) {
-                minMax = "Min";
-                increaseDecrease = "increase";
-            } else if (argNameString.includes("max")) {
-                minMax = "Max";
-                increaseDecrease = "reduce";
-            }
+            let rankedUnranked = rankedUnrankedGamesString(argNameString);
+            let minmaxList = minmaxStringList(argNameString);
+            // ["minimum" or "maximum", "increase" or "decrease", "low" or "high"]
 
             // then, specific messages for handicaponly and evenonly messages first
-            if (notification.handicap === 0 && minMax === "Min" && config[argNameString] > 0) {
+            if (notification.handicap === 0 && minmaxList[0] === "minimum" && config[argNameString] > 0) {
```

Comment:
The check `minmaxList[0] === "minimum"` is terrible. Consider returning a boolean from the `minmaxStringList` function too. That way you can do `minmax.isMin`.

That would mean you need to extend the object i described below to `{min: "minimum", inc: "increase", low: "low", isMin: true}`.

Also fix this everywhere, i see this a bunch of times.

Reply:
i see, indeed it is cleaner than if (A == true) !
thanks !